features/step_definitions/_given_steps.rb

```diff
@@ -209,15 +209,15 @@
 Given /^the [pP]roject(?: "([^\"]*)")? has the following work_packages:$/ do |project_name, table|
   project = get_project(project_name)
 
-  User.current = User.find(:first)
+  User.current = User.first
 
   as_admin do
     table.hashes.each do |task|
-      parent = WorkPackage.find(:first, conditions: { subject: task['parent'] })
-      type = Type.find_by_name(task['type'])
+      parent = WorkPackage.find_by(subject: task['parent'])
+      type = Type.find_by(name: task['type'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.